modin/core/storage_formats/base/query_compiler.py

```diff
@@ -1449,6 +1513,32 @@ def replace(self, **kwargs):  # noqa: PR02
         """
         return DataFrameDefault.register(pandas.DataFrame.replace)(self, **kwargs)
 
+    @doc_utils.add_refer_to("Series.argsort")
+    def argsort(self, **kwargs):  # noqa: PR02
+        """
+        Return the integer indices that would sort the Series values.
```

Comment:
Minor nit but why have we added a docstring here?

Reply:
Our doc checker insists on having docstrings, so we'll have to either silence it with pragmas or fill in at least some docstrings... and for methods which are a direct match for pandas I feel it useful to copy their docstrings for clarity.